### ISSUE TITLE
Add simple user interactions to the map

### ISSUE THREAD

**Issue by mhines-usgs** [usgs-makerspace/wbeep-viz#11]
Add things that average users expect to see on interactive maps. A la 
![image](https://user-images.githubusercontent.com/13220910/60907641-b2540180-a23f-11e9-9024-8f5a263cea70.png)

Could do "zoom to" features as well.

**Comment by abriggs-usgs** [comment]
Used the power of Mapbox-gl to add fullscreen toggle, increase/decrease zoom, and jump to location buttons.

Pull Request: https://github.com/usgs-makerspace/wbeep-viz/pull/16

**Comment by lindsayplatt** [comment]
Just waiting for this to get pushed up to dev.

**Comment by abriggs-usgs** [comment]
In the process of making the layer toggles, I changed the application to use the vue-mapbox plugin  which interactions with Mapbox to occur in a more component-like way. See https://soal.github.io/vue-mapbox/ for more details on the plugin. 

Currently there are toggles for all layers and there is only very basic styling applied to the buttons controlling the toggling of the layers.  This is to allow easy viewing of the various layers during development. The number of buttons and the styling should be adjusted later. 

The code for the map styles has been moved from the 'mapbox' component into a file called 'mapStyles.js' in the src/assets/mapStyles folder.

Additionally, the code is written so that any changes to the layers (such as adding additional layers) will alter the buttons without any change of the Vue component code.  This is controlled by the addition of an attribute to the map styles code. The attribute is a Boolean called 'showButton.' Not surprisingly, if the value of the 'showButton' key is set to true, the button will show. The button element will not be created when 'showButton' is set to false.
